chore: commented out PR template sections (#1412)
flame-engine/flame: .github/pull_request_template.md

## Breaking Change

<!-- Does your PR require Flame users to manually update their apps to accommodate your change? 

> If the PR is a breaking change it should be indicated with a suffixing !. For example, `feat!:`, `fix!:`). As in [Conventional Commit].

Reviewer: ```suggestion
If the PR is a breaking change this should be indicated with suffix "!"  (for example, `feat!:`, `fix!:`). See [Conventional Commit] for details.
```
